tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergOperations.java

```diff
@@ -176,8 +176,9 @@ public String transform(String transformation) {
     return transformation;
   }
 
-  public void disguice(@ParameterGroup(name = "currentLook") BarberPreferences currentLook,
-                       @ParameterGroup(name = "disguise", showInDsl = true) BarberPreferences disguise) {
+  public void disguice(@ParameterGroup(name = "currentLook") @DisplayName("Current look") BarberPreferences currentLook,
```

Comment:
The root cause of this ticket was to allow repeat param group display name, but I didn't see any repeated displayname testing if that works correctly.